### PR TITLE
add documentation & rename a class

### PR DESCRIPTION
**Pull request by CafeKrem**
add documentation for :
* redisConnectionPool + example method to use it 
* rename RsRedis into RsWrapperRedis

### CODE REVIEW

**Comment by mumez** [comment]
Thank you for the documentation. 
I actually still prefer the name 'RsRedis'. As a facade, it provides a higher level of interface, and users do not care about some internals (wrapping a RediStick, etc).

So if you choose a renaming route, my suggestion is: 
- RsRedis -rename-> RsStickyRedis
- RsRedisProxy -rename-> RsConnectionPooledRedis
- Add RsRedis as a factory of RsStickyRedis or RsConnectionPooledRedis

For example:
`RsRedis pooled default` => returns a RsConnectionPooledRedis(key: #default).
`RsRedis sticky url: 'sync://192.168.1.7'` => returns a RsStickyRedis( wrapping a stick connected to: '192.168.1.7').

What do you think?

**Comment by CafeKrem** [comment]
your wright is more like a facade, my bad. But about RsRedisWrapper/RsRedis it looks redundant maybe we should remove it and RsConnectionPool should instantiate a RsStick , what do you think?  Redis has a very simple and documented API I think we don't need to add another API.

Sorry I'm still learning I'm in first years of master degree, in an internship

**Comment by CafeKrem** [comment]
sorry I was thinking that this commit will be add to my local repo

**Comment by mumez** [comment]
My thinking for the present:
RsStickyRedis and RsConnectionPooledRedis should have the same interfaces and normally users would not care about which is used. The just instantiate one of them via RsRedis factory.

It would be simpler if we just remove RsStickyRedis, but I still think that connection pool is not necessary for most simple Redis use cases. So I would like to leave the options.

File: repository/RediStick-Core.package/RsRedisEndpoint.class/instance/keysValueInline..st [hunk 1]
```diff
@@ -0,0 +1,10 @@
+private
+keysValueInline: aDictionary
```
Comment (mumez) [inline repository/RediStick-Core.package/RsRedisEndpoint.class/instance/keysValueInline..st:2]: `keysAndValuesInlined: ` would be a better name

Comment (CafeKrem) [reply repository/RediStick-Core.package/RsRedisEndpoint.class/instance/keysValueInline..st]: done https://github.com/mumez/RediStick/pull/8/commits/e08b474a827594658cfb99de920d091273bb2e30

**Comment by CafeKrem** [comment]
for conflict, file keep your version. 
I don't know how those files have been deleted.

**Comment by mumez** [comment]
I resolved the conflicts and merged your PR to 'develop' branch, not 'master',  just to keep the commit log clean.
https://github.com/mumez/RediStick/commit/b2d4323e7eb480e2a04246e405d2c83cc6c32a63

Thank you for the contribution!

**Pull request closed without merging**